products/amazon-glue.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Amazon Glue
+category: service
+tags: amazon glue
+iconSlug: amazonaws
+permalink: /amazon-glue
+releasePolicyLink:
+  https://docs.aws.amazon.com/glue/latest/dg/glue-version-support-policy.html
+releaseDateColumn: true
+releaseColumn: false
+
+auto:
+-   custom: true
+
+# Versions taken from https://docs.aws.amazon.com/glue/latest/dg/release-notes.html
+releases:
+-   releaseCycle: "4.0"
+    releaseDate: 2022-11-28
+    eol: false
+    pythonVersion: "3.10"
+    sparkVersion: "3.3"
+
+-   releaseCycle: "3.0"
+    releaseDate: 2021-08-19
+    eol: false
+    pythonVersion: "3.7"
+    sparkVersion: "3.1"
+
+-   releaseCycle: "2.0"
+    releaseDate: 2020-08-10
+    eol: 2024-01-31
+    pythonVersion: "3.7"
+    sparkVersion: "2.4"
+
```

Comment:
Would be great to also document AWS Glue 1.0, WDYT ? It may be a bit tricky due to the support of both Python 2.7 and 3.6 though.

Reply:
@marcwrobel I just added what I think to be an okay solution to versions prior to 2.0. Let me know what you think. I also updated the description with new links for version release dates.

Reply:
I will make a small change to use `releaseLabel` instead and set a `releaseCycle` more suitable for API usage, but LGTM.